sweble-wikitext-components-parent/swc-engine/src/main/java/org/sweble/wikitext/engine/utils/LanguageConfigGenerator.java

```diff
@@ -203,10 +199,10 @@ else if (postOptional)
 			{
 				wikiConfig.addI18nAlias(i18nAlias);
 			}
-			catch (Exception e)
+			catch (IllegalArgumentException e)
 			{
 				// TODO resolve conflicts problem
-				e.printStackTrace();
+				System.out.println(e.getMessage());
```

Comment:
@tgalery Does _Sweble_ use a logging framework somewhere? It might be better suited to have SLF4J here, instead of logging to the standard console. With the above approach, the output won't be seen in Application Container environments.

Reply:
Gonna double check, and if not could introduce. My worry would be that if this is the case, the aim of this PR could become much bigger than expected. But will have a think!

Reply:
Well, not the aim of this PR, just in case there is a logger available.